src/api/constants.ts

```diff
@@ -10,15 +10,3 @@ export const CORS_HEADERS = {
         yield { 'Access-Control-Max-Age': '86400' }
     }
 }
-
-export const AUTH0_DOMAIN = _AUTH0_DOMAIN
```

Comment:
@ankurdotb @benyam7 this can't be removed, we need these environment vars. The declare block only tells the compiler that something with that name will be injected into system or it has a meaning in a way.